content/simulator/programming/_index.md

```diff
@@ -37,7 +37,7 @@ There is another robot in the arena, in its starting zone. By default, this robo
 
 To control the other robot, you will need to provide its code:
 
-Create the robot code in `zone-1/robot.py`. Your code at `robot.py` will be used for zone 0, though you are encouraged to move your code to `zone-0/robot.py` when running multiple robots. Robots which do not have any robot code in their `zone-x` directory will not do anything.
+Create the robot code in `zone-1/forklift.py` and `zone-1/crane.py`. Your original code will still be used for zone 0, though you are encouraged to move your code to `zone-0/` when running multiple robots. Robots which do not have any robot code in their `zone-x` directory will not do anything.
```

Comment:
the `zone-0` folder is created by default, so their files will probably already be in there

Reply:
I don't think it is is it? I think it just creates the in the root

Reply:
it created it in `zone-0` for me